shell/src/main/java/alluxio/shell/command/AbstractShellCommand.java

```diff
@@ -49,14 +49,7 @@ protected AbstractShellCommand(FileSystem fs) {
    * @param args the arguments for the command, excluding the command name and options
    * @return whether the args are valid
    */
-  protected boolean validateArgs(String... args) {
```

Comment:
After read the code again, I think there is a simpler way to accomplish this JIRA with less change introduced: 
- we can simply leave the this `AbstractShellCommand#validataArgs` unchanged.
- For subclass of `AbstractShellCommand` that we want to take non-fixed number of args, we could simply override their `validateArgs` to be the following, like what you did in `CatCommand`:
```java
@Override
public boolean validateArgs(String... args) { return args.length >= 1; }
```